src/command_modules/azure-cli-network/azure/cli/command_modules/network/tests/test_network_commands.py

```diff
@@ -1013,50 +1013,42 @@ class NetworkVpnGatewayScenarioTest(ResourceGroupVCRTestBase):
-        self.cmd('network vnet create -g {} -n {} --subnet-name GatewaySubnet --address-prefix 10.0.0.0/16 --subnet-prefix 10.0.0.0/24'.format(rg, vnet1_name))
-        self.cmd('network vnet create -g {} -n {} --subnet-name GatewaySubnet --address-prefix 10.1.0.0/16 --subnet-prefix 10.1.0.0/24'.format(rg, vnet2_name))
-
-        subnet1_id = '/subscriptions/{subscription_id}/resourceGroups' \
-            '/{rg}/providers/Microsoft.Network/virtualNetworks/{vnet1}/subnets/GatewaySubnet' \
-            .format(subscription_id=subscription_id, rg=rg, vnet1=vnet1_name)
-        subnet2_id = '/subscriptions/{subscription_id}/resourceGroups' \
-            '/{rg}/providers/Microsoft.Network/virtualNetworks/{vnet2}/subnets/GatewaySubnet' \
-            .format(subscription_id=subscription_id, rg=rg, vnet2=vnet2_name)
-
-        self.cmd('network vpn-gateway create -g {0} -n {1} --subnet-id {2} --public-ip-address {3}'
-                 .format(rg, gateway1_name, subnet1_id, ip1_name))
-        self.cmd('network vpn-gateway create -g {0} -n {1} --subnet-id {2} --public-ip-address {3}'
-                 .format(rg, gateway2_name, subnet2_id, ip2_name))
-
-        gateway1_id = '/subscriptions/{subscription_id}/resourceGroups' \
-            '/{rg}/providers/Microsoft.Network/virtualNetworkGateways/{gateway1_name}' \
-            .format(subscription_id=subscription_id, rg=rg, gateway1_name=gateway1_name)
-        gateway2_id = '/subscriptions/{subscription_id}/resourceGroups' \
-            '/{rg}/providers/Microsoft.Network/virtualNetworkGateways/{gateway2_name}' \
-            .format(subscription_id=subscription_id, rg=rg, gateway2_name=gateway2_name)
-
-        self.cmd('network vpn-connection create -n myconnection -g {0} --shared-key 123' \
-            ' --vnet-gateway1-id {1} --vnet-gateway2-id {2}'.format(rg, gateway1_id, gateway2_id))
+        vnet1_id = '/subscriptions/{}/resourceGroups/{}/providers/Microsoft.Network/virtualNetworks/{}'.format(subscription_id, rg, self.vnet1_name)
+        vnet2_id = '/subscriptions/{}/resourceGroups/{}/providers/Microsoft.Network/virtualNetworks/{}'.format(subscription_id, rg, self.vnet2_name)
+
+        self.cmd('network vpn-gateway create -g {} -n {} --vnet {} --public-ip-address {}'.format(rg, self.gateway1_name, vnet1_id, self.ip1_name))
+        self.cmd('network vpn-gateway create -g {} -n {} --vnet {} --public-ip-address {}'.format(rg, self.gateway2_name, vnet2_id, self.ip2_name))
+
+        gateway1_id = '/subscriptions/{}/resourceGroups/{}/providers/Microsoft.Network/virtualNetworkGateways/{}'.format(subscription_id, rg, self.gateway1_name)
+        gateway2_id = '/subscriptions/{}/resourceGroups/{}/providers/Microsoft.Network/virtualNetworkGateways/{}'.format(subscription_id, rg, self.gateway2_name)
+
+        self.cmd('network vpn-connection create -n myconnection -g {} --shared-key 123 --vnet-gateway1-id {} --vnet-gateway2-id {}'.format(rg, gateway1_id, gateway2_id))
```

Comment:
In this case, the command returns nothing, so there's nothing to verify as long as it doesn't fail. I might add more of the VPN-connection commands to this test when I go through that one, but that will involve re-recording so I will hold off for now.